packages/web/src/components/CustomerCard/CustomerCard.tsx

```diff
@@ -10,20 +10,46 @@ import {
 } from '@elvia/ui'
 import { getTerm, Terms } from '@utils/getTerm'
 import { getUrlWithParams, routes } from '@routing'
-import { Customer } from '@services/customer-portal-api'
+import { Contract, Customer } from '@services/customer-portal-api'
 import * as css from './CustomerCard.module.scss'
 
 interface CustomerCardProps {
   customer: Customer
+  contracts: Contract[]
 }
 
-export const CustomerCard: React.FC<CustomerCardProps> = ({ customer }) => {
+export const CustomerCard: React.FC<CustomerCardProps> = ({
+  customer,
+  contracts,
+}) => {
   const isDesktop = useMedia(MediaTypes.DESKTOP)
 
+  const customerContracts =
+    contracts !== null
+      ? contracts.filter(c => c.customerId === customer.id)
+      : null
```

Comment:
```suggestion
  const customerContracts = contracts?.filter(c => c.customerId === customer.id)
```
Here the optional chaining operator `?.` can be used to achieve the same effect more conciesly